openreview/venue_request/venue_request.py

```diff
@@ -1292,3 +1418,132 @@ def setup_matching(self):
+                    },
+                    'signatures': {
+                        'values-regex': '~.*|{}'.format(self.support_group.id)
+                    },
+                    'content': matching_status_content
+                }
+            ))
+
+    def setup_error_status(self):
+
+        with open(self.error_status_process, 'r') as f:
+            file_content = f.read()
+            file_content = file_content.replace("GROUP_PREFIX = ''", "GROUP_PREFIX = '" + self.super_user + "'")
+
+            self.comment_super_invitation = self.client.post_invitation(openreview.Invitation(
+                id=self.support_group.id + '/-/Stage_Error_Status',
+                readers=['everyone'],
+                writers=[self.support_group.id],
+                signatures=[self.support_group.id],
+                invitees=['everyone'],
+                process_string=file_content,
+                reply={
+                    'forum': None,
+                    'replyto': None,
+                    'readers': {
+                        'description': 'Select all user groups that should be able to read this comment.',
+                        'values': [self.support_group.id]
+                    },
+                    'writers': {
+                        'values-copied': [
+                            '{signatures}'
+                        ]
+                    },
+                    'signatures': {
+                        'values-regex': '~.*|' + self.support_group.id,
+                        'description': 'How your identity will be displayed.'
+                    },
+                    'content': {
+                        'title': {
+                            'order': 1,
+                            'value-regex': '.{1,500}',
+                            'description': 'Failed Invitation/Stage Name',
+                            'required': True
+                        },
+                        'error': {
+                            'order': 2,
+                            'value-regex': '[\\S\\s]{1,200000}',
+                            'description': 'Brief summary of the error.',
+                            'required': True,
+                            'markdown': True
+                        },
+                        'comment': {
+                            'order': 3,
+                            'value-regex': '[\\S\\s]{1,200000}',
+                            'description': 'Error description (max 200000 characters).',
+                            'required': False,
+                            'markdown': True
+                        },
+                        'reference_url': {
+                            'order': 4,
+                            'value-regex': '.{1,500}',
+                            'description': 'URL to check references',
+                            'required': False,
+                            'hidden': True
+                        },
+                        'stage_name': {
```

Comment:
If you are going to put the stage name in the title, I don't think this field is needed.

Reply:
I added this so that we can have flexibility with the Title. I can change it back.

Reply:
ok, let's keep it then.